catalog/confluence/catalog-confluence-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -48,6 +48,13 @@
         <AD name="Additional Attributes" id="additionalAttributes" required="false"
             type="String" cardinality="100"
             description="Additional attributes to add to Confluence metacards returned from this source."/>
+        <AD name="Body Expansion Field" id="bodyExpansion" required="false"
```

Comment:
❓ should this drive any documentation changes or would auto-generating the blueprint be sufficient?